lib/config/types.ts

```diff
@@ -381,6 +381,7 @@ export interface PackageRule
   matchRepositories?: string[];
   matchSourceUrls?: string[];
   matchUpdateTypes?: UpdateType[];
+  matchJsonata?: string;
```

Comment:
IMO an array would be benefitial here by allowing user to break the logic in separat elements.

Reply:
Do you mean instead of using "and"s?

Reply:
I mean OR chains

Reply:
Doesn't this then make it harder to debug eg in the playground?

Maybe we can start this way with single string and see if its really a problem

Reply:
That way it is possible to test each element independently.  

But fine for me, it would be easy to migrate configs.